packages/model-viewer/src/features/staging.ts

```diff
@@ -80,6 +79,7 @@ export const StagingMixin = <T extends Constructor<ModelViewerElementBase>>(
       this[$autoRotateStartTime] = performance.now();
     }
 
+    // tslint:disable-next-line:no-any
```

Comment:
Let's skip this. In fact, you could probably just revert this file, since there aren't any real changes here.

Reply:
Done.